src/styles/components/_testimonial.scss

```diff
@@ -0,0 +1,224 @@
+.testimonial {
```

Comment:
I like the custom properties idea for fake variables. Just please do not use PX us Rems, that why I send you guys the vs code extension to convert px to rems.